server.js

```diff
@@ -6,17 +6,49 @@ const io = require('socket.io')(http);
 const port = 5000;
 const host = 'localhost';
 
+var names = [];
+var words = [];
+var players = [];
+
+//TODO: User this object to print final point, and give points do the righ user
+var user = {name: null, word: null, points: 0}
+
 app.use(express.static(__dirname + '/public'));
+app.set('views', __dirname + '/public/views');
+app.engine('html', require('ejs').renderFile);
+app.set('view engine', 'html');
 
-//History with drawed lines to render on new client connection
-const drawedList = [];
+app.get('/draw', (req, res) => {
+  if (!(names.indexOf(req.query.name) > -1)) {
+    user.name = req.query.name
+    user.word = req.query.word
+    players.push(user);
+    names.push(req.query.name);
+    words.push(req.query.word);
+    res.render('draw', user);
+  } else{
+    //TODO: error message
+    res.redirect('/');
+  }
+})
 
 io.on('error', e => console.log(e));
 
 io.on('connection', function (socket) {
   socket.on('draw', function (line) {
     socket.broadcast.emit('draw', line);
-  });
+  })
+
+  socket.on('send word', function(obj, callback){
```

Comment:
You could use an express endpoint to receive this word and not traffic with socket
After you receive this word you could compare to players word sended and emit points if is match
You could emit event to display message in global chat after that as well.